Design-Docs/glossary.md

```diff
@@ -0,0 +1,143 @@
+---
+title: SCS Definitions (Draft)
+version: 2021-09-09-001
+authors: Matthias Hamm
+state: v1.0 (for R0)
+---
+
+Glossary - Naming conventions for SCS (Draft) 
+
+# Motivation
+
+In order to have a consistent naming scheme in SCS environments, we need a definition of the different terms. 
+
+# Proposal
+
+This is a draft version, which is free to be discussed
+
+## Definition of a Region
+
+An OpenStack/SCS Region consists of at least one or more Availability Zones that share a Control Plane with their services. As a result, they share an API, as well as a CEPH cluster, i.e. a data store. Within the region, any Layer 2 networks are available to the user.
+
```

Comment:
Does it make sense to share one or more ceph clusters? What for?

In this concept, l2 network is stretched in the region, does that make sense?